### DISCUSSION TITLE
Changing layout on My Account -> Licenses page

### DISCUSSION THREAD

**Discussion by niksen** [gdarko/digital-license-manager#16] [Ideas]
Hello!

It would be nice to have a possibility to change layout on the My Account -> Licenses page.

Currently we reviewing the code and what we want to achieve is that on the My Account -> Licenses page the structure would be: for each order you have a table with the related keys. For now the only way to change the layout is to rewrite the logic by ourselves.

We believe that this would be useful for others. Because different businesses need different layout. In our case the current layout (tables for each product) does not really fit our needs.

So if there is an option to choose the layout, it would really help.

**Comment by gdarko** [comment]
This will be considered in one of our future releases. Thanks for the suggestions @niksen .
